In the Nix sandbox, network access is not avaliable. Luckily tinycbor is avaliable with the rev
requested in nixpkgs. So this patch just allows for cmake to grab the pkgconfig module.

--- a/c2rust-ast-exporter/src/CMakeLists.txt
+++ b/c2rust-ast-exporter/src/CMakeLists.txt
@@ -6,43 +6,8 @@ project(ASTExporter)
 # TinyCBOR                                      #
 #################################################
 
-set(TINYCBOR_REPO "https://github.com/intel/tinycbor.git" CACHE STRING "tinycbor git repo URL")
-
-# v0.6.3 tag, but using the commit hash instead (of the tarball hash) for integrity checks
-# unlike a .tar.gz MD5 hash, this SHA-1 commit hash should stay stable regardless of compression/archiving
-# (GitHub has changed this), and still retains the integrity check
-set(TINYCBOR_TAG "d393c16f3eb30d0c47e6f9d92db62272f0ec4dc7" CACHE STRING "tinycbor git tag/branch/commit hash")
-
-set(TINYCBOR_PREFIX "${CMAKE_BINARY_DIR}/tinycbor" CACHE STRING "tinycbor install prefix")
-
-if(CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
-    set(MAKE "gmake")
-else()
-    set(MAKE "make")
-endif()
-
-include(ExternalProject)
-ExternalProject_Add(tinycbor_build
-            PREFIX ${TINYCBOR_PREFIX}
-            INSTALL_DIR ${CMAKE_BINARY_DIR}
-            GIT_REPOSITORY ${TINYCBOR_REPO}
-            GIT_TAG ${TINYCBOR_TAG}
-            # the fd redirection here fails when the build run inside Cargo.
-            # patch from upstream:
-            # https://github.com/intel/tinycbor/commit/6176e0a28d7c5ef3a5e9cbd02521999c412de72c
-            PATCH_COMMAND patch --forward -p1 < ${CMAKE_CURRENT_SOURCE_DIR}/tinycbor_fix_build.patch || true
-            CONFIGURE_COMMAND ${MAKE} .config && cat ${CMAKE_CURRENT_SOURCE_DIR}/tinycbor.config >> .config
-            BUILD_COMMAND ${MAKE} --quiet prefix=<INSTALL_DIR> CFLAGS=-fPIC
-            INSTALL_COMMAND ${MAKE} --quiet prefix=<INSTALL_DIR> install
-            BUILD_IN_SOURCE 1
-            BUILD_BYPRODUCTS ${CMAKE_BINARY_DIR}/lib/libtinycbor.a
-)
-
-include_directories(${CMAKE_BINARY_DIR}/include)
-
-add_library(tinycbor STATIC IMPORTED)
-set_target_properties(tinycbor PROPERTIES IMPORTED_LOCATION ${CMAKE_BINARY_DIR}/lib/libtinycbor.a)
-add_dependencies(tinycbor tinycbor_build)
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(TINYCBOR REQUIRED tinycbor)
 
 set(AST_EXPORTER_SRCS
   AstExporter.cpp
